frontend/src/components/Navigation/_Footer.scss

```diff
@@ -0,0 +1,90 @@
+@use "../../components/Basics/" as *;
+
+.footer-nav {
+  background-color: $color-blue-darker;
+  height: 128px;
+  align-items: center;
+  @include breakpoint-media-max("lgtablet") {
+    height: 226px;
+    flex-direction: column;
+    justify-content: space-evenly;
+  }
+}
+
+.logo-desktop-footer {
+  @include breakpoint-media-max("mobile") {
+    @include hidden;
+  }
+}
+
+.logo-mobile-footer {
+  @include hidden;
+  @include breakpoint-media-max("mobile") {
+    display: block;
+  }
+}
+
+.footer-icons-on-dark {
+  margin-left: 176px;
+  @include breakpoint-media-max("lgtablet") {
+    margin-left: 0;
+    margin-bottom: -17px;
+  }
+}
+
+.footer-menu {
+  @include breakpoint-media-max("lgtablet") {
+    width: 220px;
+    order: 3;
```

Comment:
Nitpick: because of the way order works (default is 0), this can be set to 1 and still work the same way. Feel free to ignore this since it is not a big deal.